README.md

```diff
@@ -1,14 +1,14 @@
-# Transformers Reading Group
+# _Robots in Disguise_: Foundation Models Reading Group
 
-Public repo for the [Research Engineering Team](https://www.turing.ac.uk/research-engineering)'s reading group on Transformers.
+Public repo for The Alan Turing Institute's reading group on foundation models.
```

Comment:
maybe add the word Foundational AI somewhere, like "as part of the activities of Foundational AI"?